src/main/scala/com/mozilla/telemetry/streaming/ExperimentEnrollmentsAggregator.scala

```diff
@@ -99,23 +99,25 @@ object ExperimentEnrollmentsAggregator extends StreamingJobBase {
     import spark.sql.functions._
 
     val events: Dataset[ExperimentEnrollmentEvent] = messages.flatMap(v => {
-      val m = Message.parseFrom(v.get(0).asInstanceOf[Array[Byte]])
-      val fields = m.fieldsAsMap
-      val docType = fields.getOrElse("docType", "").asInstanceOf[String]
-      if (!allowedDocTypes.contains(docType)) {
-        Array.empty[ExperimentEnrollmentEvent]
-      } else {
-        try {
+      try {
+        val m = Message.parseFrom(v.get(0).asInstanceOf[Array[Byte]])
+        val fields = m.fieldsAsMap
+        // TODO: looks like we can get a double here
```

Comment:
What's the actual TODO? investigate this more?

Reply:
Yes, it's for further investigation.